### PR TITLE
chore:fix all typos

### PR DESCRIPTION
**Pull request by AdventureSeeker987**
close:https://github.com/Plonky3/Plonky3/issues/343

### CODE REVIEW

File: challenger/src/duplex_challenger.rs [hunk 1]
```diff
@@ -106,8 +106,8 @@ where
     F: Copy,
     P: CryptographicPermutation<[F; WIDTH]>,
 {
-    fn observe(&mut self, valuess: Vec<Vec<F>>) {
-        for values in valuess {
+    fn observe(&mut self, values: Vec<Vec<F>>) {
+        for values in values {
```
Comment (jorgeantonio21) [inline challenger/src/duplex_challenger.rs:110]: Please do not name the item variable the same as the container variable name

File: challenger/src/multi_field_challenger.rs [hunk 2]
```diff
@@ -140,8 +140,8 @@ where
     PF: PrimeField,
     P: CryptographicPermutation<[PF; WIDTH]>,
 {
-    fn observe(&mut self, valuess: Vec<Vec<F>>) {
-        for values in valuess {
+    fn observe(&mut self, values: Vec<Vec<F>>) {
+        for values in values {
```
Comment (jorgeantonio21) [inline challenger/src/multi_field_challenger.rs:144]: Please do not name the item variable the same as the container variable name

File: challenger/src/duplex_challenger.rs (hunk 1 above)
Comment (AdventureSeeker987) [reply challenger/src/duplex_challenger.rs]: thks to point out, fixed

Comment (AdventureSeeker987) [reply challenger/src/duplex_challenger.rs]: @jorgeantonio21 Done

**Review by SyxtonPrime — APPROVED** [review]

**Pull request merged**